c7n/output.py

```diff
@@ -323,6 +322,34 @@ def get_metadata(self):
         return res
 
 
+class JSONFormatter(logging.Formatter):
+    """
+    Convert Logs to JSON for more usable cloudwatch logs
+    https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/FilterAndPatternSyntax.html
+    """
+
+    def format(self, record):
+        json_log_object = {"type": "log",
+                           "logger": record.name,
+                           "log_time": str(datetime.utcnow()),
```

Comment:
minor rabbit hole, was curious about using the existing stdlib formatter impl it ends up being
`self.formatTime(record.created)`
along with class attribute `converter = time.gmtime` 

while the indirection is a bit annoying, with the pr atm its an extra syscall per log record (vdso linux setups not withstanding). 

probably simplest is just format and do
```
 datetime.fromtimestamp(record.created).isoformat()
```

which also fixes that str(dt) is an implicit conversion and non standard format and output.

Reply:
awesome. thanks for this!